content/sensu-go/5.19/installation/install-sensu.md

```diff
@@ -30,20 +30,29 @@ See [supported platforms][5] for more information.
 <!-- Diagram source: https://www.lucidchart.com/documents/edit/3949dde6-1bad-4f37-aa01-00a71c47a91b/0 -->
 
 The **Sensu backend** gives you flexible, automated workflows to route metrics and alerts.
-It is powered by an an embedded transport and [etcd][16] datastore and 
-Sensu backends require persistent storage for their embedded database, disk space for local asset caching, and three exposed ports:
+It is powered by an an embedded transport and [etcd][16] datastore and gives you flexible, automated workflows to route metrics and alerts.
+Sensu backends require persistent storage for their embedded database, disk space for local asset caching, and several exposed ports:
 
-- `3000` - Sensu [web UI][3]
-- `8080` - Sensu [API][26] used by sensuctl, some plugins, and any of your custom tooling
-- `8081` - WebSocket API used by Sensu agents
-
-Sensu backends running in a [clustered configuration][22] require additional ports.
-See [Deploy Sensu][31] and [hardware requirements][25] for deployment recommendations.
+- 2379 (HTTP/HTTPS) Sensu storage client: Required for Sensu backends using an external etcd instance
```

Comment:
I think this content might be more easily digested when formatted as a table with headings like `port`, `protocol` and `comment`